inc/save.php

```diff
@@ -170,8 +170,8 @@ function validate_line( $line, $line_number ) {
 						'type'    => 'warning',
 						'message' => sprintf(
 							/* translators: %s: TAG-ID */
-							__( '"%s" does not appear to be a valid TAG-ID', 'ads-txt' ),
-							esc_html( $fields[3] )
+							esc_html__( '"%s" does not appear to be a valid TAG-ID', 'ads-txt' ),
```

Comment:
Same issue with escaping the double quotes.

Reply:
good catch @helen - I missed this issue in my testing. @philipjohn - can we move the escaping to the values used by sprintf in each of these cases?